.github/workflows/post-release.yml

```diff
@@ -0,0 +1,33 @@
+name: "Post release tasks"
+
+run-name: >
+  ${{ format('Post Release Tasks for Release: {0}', github.ref_name) }}
+
+on:
+  release:
+    types: [released]
+
+permissions:
+  contents: write
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  create-pkg-tags:
+    name: Create package tags
+    runs-on: ubuntu-latest
```

Comment:
should this be ubuntu-24.04?

Reply:
It should -- I was going to fix it after the other PR is merged and I can rebase this branch.

Reply:
Done!